commit 0ab3825f250486453892e3e18a702a44538bff6d
Author: Justus Winter <justus@gnupg.org>
Date:   Sun May 22 00:43:19 2016 +0200

    libdiskfs: fix error handling
    
    * libdiskfs/dir-rmdir.c (diskfs_S_dir_rmdir): Initialize 'np'.

diff --git a/libdiskfs/dir-rmdir.c b/libdiskfs/dir-rmdir.c
index 83ec37b..8a29979 100644
--- a/libdiskfs/dir-rmdir.c
+++ b/libdiskfs/dir-rmdir.c
@@ -25,7 +25,7 @@ diskfs_S_dir_rmdir (struct protid *dircred,
 		    char *name)
 {
   struct node *dnp;
-  struct node *np;
+  struct node *np = NULL;
   struct dirstat *ds = alloca (diskfs_dirstat_size);
   error_t error;
 
